org.thingpedia.iot.speaker/manifest.tt

```diff
@@ -0,0 +1,61 @@
+abstract class @org.thingpedia.iot.speaker
+#_[thingpedia_name="Speaker"]
+#_[thingpedia_description="Interface for Speaker."]
+#_[description="Interface for Speaker."]
+#[license="CC-0"]
+#[license_gplcompatible=true]
+#[subcategory="home"]
+{
+  monitorable query state(out state : Enum(on,off))
+  #_[canonical="switch state"]
+  #_[confirmation="the state of $__device"]
+  #_[formatted=["It is ${state}"]];
+
+  action set_power(in req power: Enum(on,off) #_[prompt="Do you want to turn it on or off?"] #_[canonical="power"])
+  #_[canonical="turn your speaker on or off"]
+  #_[confirmation="turn ${power} $__device"]
+  #[confirm=true];
+  #[doc="turn on or off the TV"];
+
+  action raise_volume()
```

Comment:
You use `volume_up` in dataset.tt. Did you lint?

Reply:
Linted and fixed in latest commit